autoload/ale/fixers/standardts.vim

```diff
@@ -0,0 +1,24 @@
+" Description: Fixing files with Standard for typescript.
```

Comment:
Fixers aren't tied to filetypes like linters are. I think what we should do instead is update the existing `standard.vim` file to optionally read the TypeScript options instead of the JavaScript ones if the filetype is `typescript`.